### PR TITLE
ISIS Powder - Beta testing bug fixes

### LINKED ISSUE #19754: ISIS Powder - Beta testing bugs
List of bugs discovered in beta testing for ISIS Powder scripts

## Priority:

- [x] PEARL: Splined vanadium only saving first and the naming incorrect (i.e. `tt:all` saves 1 spline)
- [ ] Absorption corrections are not being applied? Investigate and resolve
- [x] Tweak the docs to not use variables only parameters as it causes confusion
- [x] Add the instrument name to the cycle mapping examples in the tutorial
- [x] Rename .s1 / .s2 to .s01 / .s02
- [x] Rename partial Runs - > Intermediate Files
- [x] Tutorial / ISIS Example files. Use POLARIS runs from cycle 17/1

## Others (next release?):
- [ ] PEARL (ALL?): If the user enters the full path to their offset file it works until we go to create the spline name in `create_vanadium`
- [x] PEARL: Leave tt_*mode* spline whilst running `create_vanadium` instead of most recent spline 

## Low priority (next release):
- [ ] Remove `CryPowder` from Python scripts locations

### PR DESCRIPTION
**Pull request by DavidFair**
Description of work.
This PR fixes some bugs / niggles noticed during the beta testing of Mantid. The changes include:

- Fix for run details caching causing Pearl to overwrite the spline file regardless of `tt_mode`
- Several documentation tweaks and typo fixes
- Swap the tutorial data to use runs from the latest cycle (following feedback from POLARIS instrument scientist). This accounts for 3000 lines as we have swapped to the latest cal file.
- Append the current tt_mode when grouping and creating splined banks on PEARL (So they can view all produced data in `tt_mode` all.
- Switched POLARIS from having a `chopper_on` field to using `mode` like GEM as requested by scientists. 

**To test:**
Code review.

Fixes #19754 . (The remaining jobs will be moved to a separate issue)

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/Diffraction/isis_powder/routines/calibrate.py [hunk 1]
```diff
@@ -63,4 +63,7 @@ def _create_vanadium_splines(focused_spectra, instrument, run_details):
         mantid.SaveNexus(Filename=out_spline_van_file_path, InputWorkspace=ws, Append=append)
         append = True
     # Group for user convenience
-    mantid.GroupWorkspaces(InputWorkspaces=splined_ws_list, OutputWorkspace="Van_spline_data")
+    group_name = "Van_spline_data"
+    tt_mode = instrument._get_current_tt_mode()
+    group_name = group_name + '_' + tt_mode if tt_mode else group_name
```
Comment (PranavBahuguna) [inline scripts/Diffraction/isis_powder/routines/calibrate.py]: Admittedly a nitpick, but wouldn't the following be simpler?

```
if tt_mode:
  group_name += '_' + tt_mode
```

**Comment by PranavBahuguna** [comment]
Code I've seen so far looks good. Awaiting checks to pass.

**Comment by DavidFair** [comment]
I've got another change to make that came in the morning before we can carry on with this PR

File: docs/source/api/python/techniques/ISISPowder-Polaris-v1.rst [hunk 2]
```diff
@@ -263,25 +263,27 @@ Example Input:
...
+mode
 ^^^^^^^^^^
-The chopper state to use in the 
+The current chopper mode to use in the 
 :ref:`create_vanadium_polaris_isis-powder-diffraction-ref`
 and :ref:`focus_polaris_isis-powder-diffraction-ref` method.
 This determines which vanadium and empty run numbers
 to use whilst processing.
 
 Accepted values are: **True** or **False**
```
Comment (PranavBahuguna) [inline docs/source/api/python/techniques/ISISPowder-Polaris-v1.rst]: Shouldn't accepted values now be `PDF` or `Rietveld` now that you have changed it?

**Comment by PranavBahuguna** [comment]
Again code is alright, just see my one line comment.

File: docs/source/api/python/techniques/ISISPowder-Polaris-v1.rst (hunk 2 above)
Comment (DavidFair) [inline docs/source/api/python/techniques/ISISPowder-Polaris-v1.rst]: Good spot - Thought I had fixed up all the references, guess not.

**Comment by PranavBahuguna** [comment]
:shipit: once all checks pass

**Comment by NickDraper** [comment]
All looks good, tested and documented

**Pull request merged**